index.js

```diff
@@ -121,32 +123,42 @@ module.exports = options => {
 
       const protocol = startConfig.protocol === 'http' ? http : https;
 
-      bootstrap.server = protocol.createServer(app);
+      instance.server = protocol.createServer(app);
 
       return new Promise((resolve, reject) => {
-        bootstrap.server.listen(startConfig.port, startConfig.host, err => {
+        instance.server.listen(startConfig.port, startConfig.host, err => {
           if (err) {
             reject(new Error('Unable to connect to server'));
           }
-          resolve(bootstrap);
+          resolve(instance);
         });
       });
     },
 
     stop() {
-      return new Promise((resolve, reject) => bootstrap.server.close(err => {
+      return new Promise((resolve, reject) => instance.server.close(err => {
         if (err) {
           reject(new Error('Unable to stop server'));
         }
-        resolve(bootstrap);
+        resolve(instance);
       }));
     }
   };
 
   if (config.start !== false) {
-    bootstrap.start(config);
+    instance.start(config);
   }
 
-  return bootstrap;
+  return instance;
+
+}
 
+bootstrap.configure = function configure(key, val) {
```

Comment:
bootstrap _could_ be an es6 `class`, this could be a `static` method... discuss!